eq_translations/extractable_strings.py

```diff
@@ -239,6 +239,72 @@
         "context": "Question",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..primary_content[*].title",
+        "description": "Introduction main title",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].list[*]",
+        "description": "Introduction main list item",
+        "context": "PrimaryContent",
+        "additional_context": ["ListHeading", "ListDescription"],
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].description",
+        "description": "Introduction main description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].title",
```

Comment:
Guidance content can also contain list items. Which should also have ListHeading additional context.

Reply:
Added list items extractable strings, added test as well.